app/backend/src/couchers/servicers/communities.py

```diff
@@ -100,7 +100,7 @@ def ListCommunities(self, request, context):
             next_node_id = int(request.page_token) if request.page_token else 0
             nodes = (
                 session.query(Node)
-                .filter(Node.parent_node_id == request.community_id)
+                .filter(or_(Node.parent_node_id == request.community_id, request.community_id == 0))
```

Comment:
`(request.community_id == 0) or (Node.parent_node_id == request.community_id)` should work fine as well since the or can be evalated before the query takes place.